java/flight/flight-core/README.md

```diff
@@ -23,73 +23,4 @@ Exposing Apache Arrow data on the wire.
 
 [Protocol Description Slides](https://www.slideshare.net/JacquesNadeau5/apache-arrow-flight-overview)
 
-[GRPC Protocol Definition](https://github.com/apache/arrow/blob/master/format/Flight.proto)
-
-## Example usage
```

Comment:
this is no longer accurate?

Reply:
It's not, I removed all the old 'example' code in favor of keeping things in the cookbook; also I don't think we should be steering people towards generating Protobuf/gRPC stubs here. That said I just updated it to actually link to the cookbook.